examples/pytorch/gat/output_check.sh

```diff
@@ -0,0 +1,4 @@
+export SNARK_LIB_PATH="$(pwd)/src/cc/lib/libwrapper.so"
+python examples/pytorch/gat/main.py > log.txt
+echo "$(cat log.txt)"
+if [ "$(grep -ic '0.826 | 0.721931' log.txt)" != "1" ]; then exit 3; fi
```

Comment:
instead of copy pasting this file in every model:
1. extract common functionality
2. pass expected values as command line arguments.